news/2021-12-16-project-loved-december-2021.md

```diff
@@ -0,0 +1,523 @@
+
+[t+pazolite vs. P\*Light - IZANA](https://osu.ppy.sh/beatmapsets/1175068#fruits) by [Bunnrei](https://osu.ppy.sh/users/829284)\
+222 BPM, 2:01 | 6.45★\
+*description written by [Spectator](https://osu.ppy.sh/users/702598)*
+
+Ever played or watched osu!catch World Cup 2020? *IZANA* by Bunnrei was featured as a NoMod pick in the CWC2020 Grand Finals, which brought a huge challenge even to those who have climbed up to the final stage of the tournament. The map contains a lot of anti-flow jumps, zoom sliders with hyperdashes and stair patterns, yet Bunnrei proves once again that they are able to make them enjoyable.
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/1160712.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[Tatsh - Xevel -Platinum Long Ver.-](https://osu.ppy.sh/beatmapsets/1160712#fruits) by [Del](https://osu.ppy.sh/users/2996241)\
+176 BPM, 4:19 | 7.45★\
+*description written by [Tenshichan](https://osu.ppy.sh/users/1101600)*
+
+Del is one of those players that mapped a lot but never really tried hard enough to get their maps ranked, leaving a lot of hidden gems behind that remain untouched. *Xevel* is one of those.
+
+The difficulty of this map is rather high because of the unusual style of the map. Even though it utilizes patterns you would also see in one or the other ranked map, they are placed in a way that makes catching them difficult. Doing good on this map requires consistency, good reading and good catcher control. Certainly a map for people that are on another level.
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/1191248.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[THINK-chan - FOUR SEASONS](https://osu.ppy.sh/beatmapsets/1191248#fruits) by [Tachibana Gekka](https://osu.ppy.sh/users/4363530)\
+220 BPM, 2:15 | 6.98★, 6.98★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+We are a little late with this map being put up to the Loved voting, but better late than never!
+
+I personally didn't know any of Tachibana Gekka maps, so when I saw this map I was pleasantly surprised by how it was mapped. The map is short, the BPM is fast and the mapping style is very clean. It switches between a very normal-style map, to bursts when the music calls for it and then goes back around to the normal style. The changes blend well together, making it a very well rounded map.
+
+[![](/wiki/shared/news/2021-12-16-project-loved-december-2021/1130748.jpg)](https://osu.ppy.sh/community/forums/topics/{{TOPIC_ID}})
+
+[ExileLord - Obsidian](https://osu.ppy.sh/beatmapsets/1130748#fruits) by [Sutavv](https://osu.ppy.sh/users/5320583)\
+170 BPM, 3:17 | 10.69★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+Sutavv making their first appearance in the Loved category and it is quite an entrance if I have to say so myself.
+
+- Star rating over 10: check\
```

Comment:
Typo?

Reply:
the news post generator must've confused list items for a hatnote

Reply:
```suggestion
- Star rating over 10: check
```